main.yml

```diff
@@ -18,22 +27,59 @@ approval:
       - /ansible/catalog/approval
     reload: catalog/approval
 
+automation-analytics:
+  title: Automation Analytics
+  api:
+    versions:
+      - v1
+    isBeta: true
+  deployment_repo: https://github.com/RedHatInsights/tower-analytics-frontend-build.git
```

Comment:
does this need a `.git`?

Reply:
Not sure - it seems to work either way. For instance, Openshift uses .git with https.

Reply:
It can be left off, since `.git` indicated bare repositories http://www.saintsjd.com/2011/01/what-is-a-bare-git-repository/ but GH is smart enough and translates `.git` to working dirs. Just an insteresting thing to know.